opencog/nlp/chatbot/README

```diff
@@ -82,9 +37,13 @@ running ./opencog-server.sh from the main RelEx directory. (You must
 compile RelEx first, by installing java and ant, and saying "ant" to
 build RelEx).
 
-The parse server listens on port 4444: it will read plain-text input,
-and generates parsed text in opencog format.  The chatbot needs this
-server to process text.
+The parse server listens on port 4444: it will read plain-text input and
+generates parsed text in OpenCog format. It uses RelEx2Logic rules 
```

Comment:
Linas,

I believe that in the currently working system, R2L is still split between
some code in the RelEx output generator, and some Scheme functions.

All the stuff in the Java R2L code has been ported to Scheme functions now
(by Aaron)....  However, nobody has yet written a process to actually apply
these functions.  Mandeep is going to do this during the coming week.  But
for the moment, I believe Rodas is right that the NLP pipeline is using the
old partly-Java R2L pipeline.  Unless you changed something and I didn't
notice ;)

--ben

On Mon, Jul 13, 2015 at 1:27 AM, Linas Vepštas <notifications@github.com>
wrote:

> In opencog/nlp/chatbot/README
> <https://github.com/opencog/opencog/pull/1675#discussion_r34434183>:
>
> > @@ -82,9 +37,13 @@ running ./opencog-server.sh from the main RelEx directory. (You must
> >  compile RelEx first, by installing java and ant, and saying "ant" to
> >  build RelEx).
> >
> > -The parse server listens on port 4444: it will read plain-text input,
> > -and generates parsed text in opencog format.  The chatbot needs this
> > -server to process text.
> > +The parse server listens on port 4444: it will read plain-text input and
> > +generates parsed text in OpenCog format. It uses RelEx2Logic rules
>
> This is not quite correct; R2L iis now done in opencog, and not in relex.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog/pull/1675/files#r34434183>.
>



-- 
Ben Goertzel, PhD
http://goertzel.org

"The reasonable man adapts himself to the world: the unreasonable one
persists in trying to adapt the world to himself. Therefore all progress
depends on the unreasonable man." -- George Bernard Shaw